5m5v-bot.js

```diff
@@ -13,6 +13,14 @@ try {
   throw `Unable to load config file: ${error.message}`;
 }
 
+if (!('users' in (config ?? [])) || config.users.length === 0) {
+  throw 'No users defined in config file';
+}
+
+if (!config.users.every(user => ['language', 'email', 'username', 'password'].every(key => key in (user ?? [])))) {
+  throw 'At least one user is missing required fields in config file';
```

Comment:
I think in general it's not a bad thing to check for those fields for every user entry, but the error message should be altered. Something like "You are missing fields for one of your user entries"